recipe/meta.yaml

```
@@ -14,42 +25,95 @@ source:
     - patches/0004-add-missing-osx-workaround.patch
 
 build:
-  number: 0
-  run_exports:
-    - {{ pin_subpackage("abseil-cpp", max_pin="x.x") }}
+  number: 1
 
-requirements:
-  build:
-    - {{ compiler('c') }}
-    - {{ compiler('cxx') }}
-    - cmake
-    - ninja
+outputs:
+  {% if shared_libs == "ON" %}
+  - name: libabseil_shared
+    script: build-abseil.sh   # [unix]
+    build:
+      string: cxx{{ cxx_standard }}_h{{ PKG_HASH }}_{{ PKG_BUILDNUM }}
+      run_exports:
+        - libabseil_shared ={{ version }}=cxx{{ cxx_standard }}*
+    requirements:
+      build:
+        - {{ compiler('c') }}
+        - {{ compiler('cxx') }}
+        - cmake
+        - ninja
+    test:
+      commands:
+        {% for each_lib in absl_libs %}
+          # presence of shared libs
+          - test -f $PREFIX/lib/libabsl_{{ each_lib }}${SHLIB_EXT}
+          # absence of static libs
+          - test ! -f $PREFIX/lib/libabsl_{{ each_lib }}.a
+        {% endfor %}
 
-{% set absl_libs = [
-    "base", "civil_time", "cord", "cordz_functions", "cordz_handle", "cordz_info",
-    "cordz_sample_token", "examine_stack", "exponential_biased", "failure_signal_handler",
-    "flags", "flags_commandlineflag", "flags_config", "flags_marshalling", "flags_parse",
-    "flags_private_handle_accessor", "flags_program_name", "flags_reflection", "flags_usage",
-    "hash", "hashtablez_sampler", "int128", "log_severity", "low_level_hash", "periodic_sampler",
-    "random_distributions", "random_seed_gen_exception", "random_seed_sequences", "raw_hash_set",
-    "scoped_set_env", "spinlock_wait", "stacktrace", "status", "statusor", "strerror", "strings",
-    "symbolize", "synchronization", "time", "time_zone"
-] %}
-test:
-  commands:
-    # windows (DLL + import library)
-    - if not exist %LIBRARY_BIN%\\abseil_dll.dll exit 1  # [win]
-    - if not exist %LIBRARY_LIB%\\abseil_dll.lib exit 1  # [win]
+  {% else %}  # shared_libs == "OFF"
+
+  - name: libabseil_static
```

```suggestion
  - name: libabseil-static
```
package names should use "-" instead of "_".

https://github.com/conda-forge/cfep/blob/main/cfep-18.md